src/modules/shared/pages/account/manage-details/manage-details-info.component.ts

```diff
@@ -66,7 +66,7 @@ export class PageAccountManageDetailsInfoComponent extends CoreComponent impleme
       this.summaryList = [
         { label: 'Name', value: user.displayName, editStepNumber: 1 },
         { label: 'Email address', value: user.email },
-        { label: 'User type', value: this.stores.authentication.getRoleDescription(user.type as 'ADMIN') }
+        { label: 'User type', value: this.stores.authentication.getRoleDescription(user.roles[0].role as 'ADMIN') }

```

Comment:
This doesn't sound good, maybe have a TODO here and/or validate. Why is the roles[0] selected in this case and not the other ones?

Reply:
Resolving this considering all the roles[0] will be reviewed in the future